static/src/js/components/TemporalDisplay/__tests__/TemporalDisplay.test.js

```diff
@@ -86,84 +89,20 @@ describe('TemporalDisplay component', () => {
     expect(screen.getByText(/2019 - 2019/i)).toBeInTheDocument()
   })
 
-  test('with the same props should not rerender', () => {
```

Comment:
This only tests the framework itself and did not change our coverage
![Uploading image.png…]()

Reply:
double check if its useful put back but, improve the test doc-string

Reply:
Putting this back its testing memoization